scripts/README.md

```diff
@@ -50,11 +58,23 @@ function teardown() {
   echo "TODO: tear down test resources"
 }
 
+function parse_flags() {
+  if [[ "$1" == "--dont-wait-for-knative" ]]; then
```

Comment:
we should call this --no-knative-wait instead? Shorter and follows the no-* convention

Reply:
It's just an example, but anyway... Done.